pkg/cloudprovider/providers/azure/azure_test.go

```diff
@@ -887,3 +985,8 @@ func TestMetadataParsing(t *testing.T) {
 		t.Errorf("Unexpected inequality:\n%#v\nvs\n%#v", network, networkJSON)
 	}
 }
+
+func addTestSubnet(svc *v1.Service) {
+	svc.Annotations[ServiceAnnotationLoadBalancerInternal] = "true"
```

Comment:
consider make use of getInternalTestService, and leave this func add subnet only.